glean_parser/templates/kotlin.buildinfo.jinja2

```diff
@@ -0,0 +1,25 @@
+// -*- mode: kotlin -*-
+
+/*
+ * AUTOGENERATED BY glean_parser.  DO NOT EDIT.
+ */
+{# The rendered markdown is autogenerated, but this
+Jinja2 template is not. Please file bugs! #}
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package {{ namespace }}
+
+import {{ glean_namespace }}.BuildInfo
+import {{ namespace_package }}.BuildConfig
+
+internal object GleanBuildInfo {
+    val buildInfo: BuildInfo by lazy {
+        BuildInfo(
+            BuildConfig.VERSION_CODE.toString(),
```

Comment:
should this use explicitly named parameters? Just so we don't get confused when we add more arguments eventually?

Reply:
Good idea.